### ISSUE TITLE
Elevated circulating o-Phosphoethanolamine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6044]
**Preferred term label:** Elevated circulating o-Phosphoethanolamine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of o-Phosphoethanolamine in the blood is above the normal range., PMID: 29342854

**Parent term (use hpo.jax.org/app)** Increased circulating ethanolamine concentration HP:0500250

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I am wondering if the current term https://hpo.jax.org/app/browse/term/HP:0500250 actually is Phosphoethanolaminuria -- there are no associated diseases and I do not find a clear clinical distinction between Ethanolaminuria and Phosphoethanolaminuria. I find only a LOINC test for Phosphoethanolaminuria. @JillMiller  -- can you provide additional context?

**Comment by pnrobinson** [comment]
adding term
